### PR TITLE
Builder

### PR DESCRIPTION
**Pull request by Lennaayy**
Builder implemented with four different concession examples.

Concession edit adapted to work with the new format.

Commented out all the metrics stuff for now.

### CODE REVIEW

**Review by LukeMaher98 — APPROVED** [review]
Looks fine

File: databases/concessions_db.txt [hunk 1]
```diff
@@ -1,8 +1 @@
-Kids Combo, 10e
-Popcorn, 7e
-Drink, 4e
-Hot Dog, 7e
-Nachos, 6e
-Chocolate Bites, 4e
-Twix, 7e
-Pizza, 15e
+popcorn,hotdog,cola,nachos,
```
Comment (Samuelpf98) [inline databases/concessions_db.txt:1]: I removed nachos from this database, then logged in as admin. I could not add the nacho concession back. "All concessions have been added" even though only three in the selector.

File: databases/concession_sales_db.txt [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
-Popcorn,102,
-Drink,85,
-Hot Dog,63,
-Nachos,110,
-Chicken Strips,85,
-Chocolate Bites,110,
-Twix,19,
-Pizza,5,
+Nachos,67,
+Popcorn,15,
```
Comment (Samuelpf98) [inline databases/concession_sales_db.txt:2]: When I purchase concessions as a user then go to the admin page it doesn't update the sales for me.

**Review by Samuelpf98 — CHANGES_REQUESTED** [review]
I found another bug where if you click the concessions basket while it has nothing in it the program crashes. Went back to a previous version and I noticed the bug has also been there so, fix if you want but you didn't cause it. Other than the two small issues it seems good.

File: databases/concession_sales_db.txt (hunk 2 above)
Comment (Lennaayy) [reply databases/concession_sales_db.txt:2]: Functionality added for sales, and similar issues noted in readme.

**Review by LukeMaher98 — APPROVED** [review]

**Pull request merged**